trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/results/ReqCheck.java

```diff
@@ -128,7 +135,15 @@ public ReqCheck merge(final ReqCheck other) {
 		final int endline = Math.max(mEndline, other.mEndline);
 		final String[] reqIds = DataStructureUtils.concat(mReqIds, other.mReqIds);
 		final String[] peaNames = DataStructureUtils.concat(mPeaNames, other.mPeaNames);
-		return new ReqCheck(newSpec, startline, endline, reqIds, peaNames);
+		final String message = mMessage.concat(other.getMessage());
+		return new ReqCheck(newSpec, startline, endline, reqIds, peaNames, message);
+	}
+	
+	private String createMessage() {
+		if(mMessage.isEmpty()) {
+			return "";
+		}

```

Comment:
If the `mMessage` is already empty, you do not need to check it again with the `if` statement. Instead, do you want to check if the message is `null` here?